packages/api-common/src/services/file.ts

```diff
@@ -156,6 +156,7 @@ export class FileService {
       expires: Date.now() + expiresIn * 1000,
       fields: {
         'Content-Type': contentType,
+        'Cache-Control': 'private, max-age=0',
```

Comment:
Please add TODO that we might change strategy about upload profile image so this line might not require anymore in the future